File: 03sim/x3p46_MC_integration/MC_pi.R
# MC Integration (R version)

N <- 1000000
count <- 0

for (i in 1:N) {
  x <- runif(1)
  y <- runif(1)
  if (x^2 + y^2 < 1){
    count <- count + 1
  }
}

4*count/N

File: 03sim/x3p46_MC_integration/MC_pi.m
% MC integration
N = 1000000;
count = 0;

for i = 1:N
    x = rand(); y=rand();
    if (x^2 + y^2) < 1
        count = count +1;
    end
end

disp(['Approximation of pi = ', num2str(4*count/N)]);
